tests/dragonfly/eval_test.py

```diff
@@ -264,3 +264,12 @@ async def test_lua_auto_async(async_client: aioredis.Redis):
 
     flushes = (await async_client.info("transaction"))["eval_squashed_flushes"]
     assert 1 <= flushes <= 3  # all 100 commands are executed in at most 3 batches
+
+
+@dfly_args({"proactor_threads": 4, "lua_auto_async": None})
```

Comment:
I prefer testing this via `interpreter_test.cc` no need for regression tests for this :)